src/test/java/org/codeseoul/event_member_management/event/EventRestIntegrationTests.java

```diff
@@ -0,0 +1,164 @@
+package org.codeseoul.event_member_management.event;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.codeseoul.event_member_management.series.Series;
+import org.codeseoul.event_member_management.series.SeriesRepository;
+import org.codeseoul.event_member_management.venue.Venue;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.sql.Timestamp;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.core.Is.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class EventRestIntegrationTests {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @Autowired
+    private SeriesRepository seriesRepository;
+
+    @Autowired
+    private EventRepository eventRepository;
+
+    private Series aSeries;
+    private Event anEvent;
+
+    @BeforeEach
+    void setUp() {
+        aSeries = new Series("New Series");
+        seriesRepository.save(aSeries);
+        anEvent = new Event(
+                "New Event",
+                "such new, so event",
+                Timestamp.valueOf("2022-06-22 08:51:00"),
+                120,
+                "http://image.com",
+                "WCoding",
+                null,
+                aSeries);
+        eventRepository.save(anEvent);
+    }
+
+    @AfterEach
+    void tearDown() {
+        eventRepository.deleteAll();
+        seriesRepository.deleteAll();
+    }
+
+    private Event createAnotherEvent() {
+        return new Event(
+                "Another Event",
+                "This is madness",
+                Timestamp.valueOf("2022-06-23 08:51:00"),
+                120,
+                "https://imgs.com",
+                "Under da sea",
+                null,
+                aSeries
+        );
+    }
+
+    private Series createAnotherSeries() {
+        return new Series("Another Series");
+    }
```

Comment:
This one is unused

Reply:
Ah yeah, I initially wanted to test changing the series for an event, but I realised the series is not part of the returned json. I'll remove it tomorrow